blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html

```diff
@@ -0,0 +1,34 @@
+<p> Based on Software Carpentry version 5.2, we translated <a href="http://www.xwmooc.net/swc/">all the lessons</a> in the form of pdf, html, mobi, epub, and azw3. It took almost 6 months: the first half spent on mostly lessons, and the second half spent on remainings. While translating and building various ebook formats, the original ebook production software of Software Carpentry seemed not to have upfront architectural concerns in terms of Chinese, Japanese, and Korean (aka, CJK)characters problems. Also, we are wondering how to generate various ebook formats from the current developing lessons. 
+</p>
+<!-- end excerpt -->
+
+<p> As the starting point of Software Carpentry in Korea, we started translating Software Carpentry lessons last October. <a href="http://software-carpentry.org/blog/2014/11/korean-translation.html">Initial draft version</a> has been released last September with the Greg and Raniere helps. The <a href="http://software-carpentry.org/blog/2014/11/korean-translation.html">first Software Carpentry</a> has been approved and held at RAPA with the only translated lessons. </p>
+
+<p> After the first workshop in Korea, we translated the remaining pieces and attempted to generate various ebook formats. Unfortunately, there were some issues mostly related to CJK problems. Still there are some issues but learners who can understand Korean can assess the Software Carpentry lessons and related materials through various formats.</p>
+
+<h3>Korean translation of Software Carpentry - Ver 5.2</a></h3>
+  <ul>
+    <ol>
+      <li> <a href="http://www.xwmooc.net/swc/book.html">HTML</a> </li>
+      <li> <a href="http://www.xwmooc.net/swc/book.pdf">PDF</a> </li>
+      <li> <a href="http://www.xwmooc.net/swc/book.epub">e-book (epub) </a> </li>
+      <li> <a href="http://www.xwmooc.net/swc/book.azw3">Amazon Kindle (AZW3) </a> </li>
+      <li> <a href="http://www.xwmooc.net/swc/book.mobi">MOBI</a> </li>
+      <li> <a href="https://github.com/statkclee/xwmooc-sc/tree/book">Software Carpentry in Korean (Ver 5.2)</a> </li>
+      <li> <a href="https://github.com/swcarpentry/bc">Software Carpentry in English (Ver 5.2)</a> </li>      
+    </ol>
+  </ul>
+
+<h3>What we found and the next...</h3>
+<p>The original ebook production software did not support CJK languages such that we need to tweak in order to make them properly rendered. We have some concerns in terms of the current lessons. We could not find any explicit commitment to the various ebook formats support. We follow other software and associated licenses, but we thought that contents license(creative commons license) need to be modified. Therefore we enforced CC-BY-SA instead of CC-BY only for Korean translated pieces. We hope many people in Korea open and share these excellent contents alike. By the way, we are excited to use fully translated materials with various formats. We look forwards to <a href="statkclee.github.io/2015-04-29-pangyo">the second workshop</a>.  
```

Comment:
Sorry for did not find time for clean my patch to generate PDF, EPUB, ... Maybe you can share yours with us. =)